ios/NewExpensifyShare/Info.plist

```diff
@@ -7,7 +7,7 @@
 	<key>ReactNativeShareMenuAppGroup</key>
 	<string>group.com.chat.expensify.chat</string>
 	<key>HostAppURLScheme</key>
-	<string>new-expensify://</string>
+	<string>new-expensify://share</string>
```

Comment:
This change has been made in #71 -- forgot to commit it before I made this branch.